jujusvg.go

```diff
@@ -10,11 +12,51 @@ import (
 	"gopkg.in/juju/charm.v5-unstable"
 )
 
-// NewFromBundle returns a new Canvas that can be used
+func NewFromBundleEmbedIcons(b *charm.BundleData, iconURL func(*charm.Reference) string) (*Canvas, error) {
```

Comment:
Scratch that, I now see what it does. It definitely needs a doc comment though.
Perhaps we could just add a bool argument to NewFromBundle (embedIcons bool) instead of adding another entry point, although I appreciate that's a backwardly incompatible change.

Alternatively, perhaps a method on Canvas might work rather than a new constructor:

    // FetchIcons fetches all the icons referenced by the canvas. They will be
    // embedded directly in the resulting SVG when it is rendered.
    func (c *Canvas) FetchIcons() error

?